sdk/eventhub/azure-eventhubs/azure/eventhub/producer.py

```diff
@@ -298,16 +302,34 @@ def _error(outcome, condition):
         if outcome != constants.MessageSendResult.Ok:
             raise condition
 
+    def create_batch(self, max_size=None, partition_key=None):
+        """
+        Create an EventDataBatch object with max message size being max_message_size.
```

Comment:
The parameter is called `max_size` not `max_message_size`

Reply:
Parameter name and description updated.